modules/units.py

```diff
@@ -0,0 +1,102 @@
+import discord
```

Comment:
Please use spaces instead of tabs across this file, in line with the rest of the code base.